bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -17,23 +17,30 @@ spec:
     # upgraded and there are duplicate rows on the "right" side of the join.
     - alert: NodeWithoutSDNController
       annotations:
-        summary: All control plane nodes should be running a sdn controller pod, {{"{{"}} $labels.node {{"}}"}} is not.
+        summary: All control plane nodes should be running an OpenShift SDN controller pod, {{"{{"}} $labels.node {{"}}"}} is not.
+        description: |
+          If at least one OpenShift SDN controller is 'Running', network control plane should be functional but
+          high availability is degraded when a controller is not functional.
       expr: |
           count(kube_node_role{role="master"} == 1) != count(kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-controller.*"})
       for: 10m
       labels:
         severity: warning
     - alert: NodeWithoutSDNPod
       annotations:
-        summary: All nodes should be running an sdn pod, {{"{{"}} $labels.node {{"}}"}} is not.
+        summary: All nodes should be running an OpenShift SDN pod, {{"{{"}} $labels.node {{"}}"}} is not.
+        description: Network control plane configuration on the node could be degraded.
       expr: |
         (kube_node_info unless on(node) topk by (node) (1, kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"})) > 0
       for: 10m
       labels:
         severity: warning
     - alert: NodeProxyApplySlow
       annotations:
-        summary: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is taking too long, on average, to apply kubernetes service rules to iptables.
+        summary: OpenShift SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is taking too long to configure kubernetes services.
```

Comment:
"kubernetes services" kind of sounds like it means "the services that make up 'kubernetes'".

Maybe something like "taking too long to update proxying rules for services" or something like that. Really, I think it's fine to mention iptables by name though